pack/fhv/fhvc.json

```diff
@@ -135,6 +135,448 @@
+    "illustrator": "Walter Brocca",
+    "name": "Western Woods",
+    "pack_code": "fhvc",
+    "position": 116,
+    "quantity": 2,
+    "shroud": 2,
+    "text": "Mushroom Grove gets +2 shroud while it is not [[Dark]].\n<b>Forced</b> - After you reveal Mushroom Grove: Test [intellect] (X), where X is the current darkness level. If you fail, the enemy in pursuit with the highest evade attacks you.",
+    "traits": "Forest. Dark.",
+    "type_code": "location"
+  },
+  {
+    "back_illustrator": "Simon Dominic",
+    "back_name": "Moonlit Clearing",
+    "back_traits": "Forest. Lair. Dark.",
+    "clues": 1,
+    "code": "10617",
+    "double_sided": true,
+    "encounter_code": "the_twisted_hollow",
+    "encounter_position": 16,
+    "faction_code": "mythos",
+    "flavor": "The trees grew too thickly, and their trunks were too big for any healthy New England wood. There was too much silence in the dim alleys between them, and the floor was too soft with the dank moss and mattings of infinite years of decay.\n - H.P. Lovecraft, \"The Colour Out of Space\"",
+    "illustrator": "Walter Brocca",
+    "name": "Western Woods",
+    "pack_code": "fhvc",
+    "position": 117,
+    "quantity": 1,
+    "shroud": 1,
+    "text": "While an enemy is moving, its location is considered to be connected to Moonlit Clearing.\n<b>Forced<\b> - After you reveal Moonlit Clearing: Test [willpower] (X), where X is the current darkness level. If you fail, find the enemy in pursuit with the highest fight and spawn it at this location.",
+    "traits": "Forest. Dark.",
+    "type_code": "location"
+  },
+  {
+    "back_flavor": "The winding paths loop back on themselves.",
+    "back_illustrator": "Walter Brocca",
+    "back_name": "Crooked Path",
+    "clues": 1,
+    "clues_fixed": true,
+    "code": "10618",
+    "double_sided": true,
+    "encounter_code": "the_twisted_hollow",
+    "encounter_position": 17,
+    "faction_code": "mythos",
+    "flavor": "The trees grew too thickly, and their trunks were too big for any healthy New England wood. There was too much silence in the dim alleys between them, and the floor was too soft with the dank moss and mattings of infinite years of decay.\n - H.P. Lovecraft, \"The Colour Out of Space\"",
+    "illustrator": "Walter Brocca",
+    "name": "Western Woods",
+    "pack_code": "fhvc",
+    "position": 118,
+    "quantity": 3,
+    "shroud": -2,
+    "text": "X is the current darkness level.\n[reaction] After you discover a clue at this location: Look at the revealed side of a connecting location. You may move to that location.",
+    "traits": "Forest. Dark.",
+    "type_code": "location"
+  },
+  {
+    "back_illustrator": "Lin Hsiang",
```

Comment:
Please add:

`"back_flavor": "The trees grew too thickly, and their trunks were too big for any healthy New England wood. There was too much silence in the dim alleys between them, and the floor was too soft with the dank moss and mattings of infinite years of decay.\n - H. P. Lovecraft, \"The Colour Out of Space\"",`